### DISCUSSION TITLE
How does Far compare to NDN?

### DISCUSSION THREAD

**Discussion by ags1234** [FarGroup/FarManager#814] [General]
It seems that NDN (Necromancer's Dos Navigator, http://ndn.muxe.com/) is also a mature project, also open-source, packed with features, but only about 10% the size on disk of that of Far.
I wonder what is the opinion of more knowledgeable users, how do these compare?

Edit: Turns out Necromancer's Dos Navigator is not open-source, but another fork of the original Dos Navigator, Dos Navigator Open Source Project, is.

**Comment by shmuz** [comment]
I've never used NDN but from the info on its site it seems it does not have plugins support and it has only very basic keyboard macro support. There are of course many other differences but I didn't look deep.

**Comment by alabuzhev** [comment]
The difference is largely philosophical.
DN has always been "everything and a kitchen sink" out of the box, while Far is more like a basic toolbox and an ecosystem that enables you to build whatever you like.
Both approaches have pros and cons and ultimately it's your call.

**Comment by shmuz** [comment]
1. I was wrong about plugins support - it does have plugins.
2. NDN.exe is 4229 KB, Far.exe is 5990 KB - it is 70% of Far.exe's size. Nowadays with huge disks it hardly matters.

**Reply by ags1234** [reply]
When I wrote that NDN is 10% of Far I was counting the directory size, not only the exe.
To compare only exe size might not be fair, since the exe needs the different other resources from the directory, otherwise they would not be there.

**Comment by zg0** [comment]
where can I download ndn sources?

**Reply by bitraid** [reply]
Necromancer's Dos Navigator is not open source [^1]. The original DOS Navigator sources are available at https://www.ritlabs.com/en/products/dn/, and there is also an updated version at https://dnosp.su/.

[^1]:http://forums.muxe.com/index.php?/topic/300-open-source-ndn/

**Reply by zg0** [reply]
I know, but topic starter states that ndn opensource. I'm very interested to know how did it happen that two sentences in initial post contains two lies?

**Comment by ags1234** [comment]
@zg0
You are right, it seems I mixed up Necromancer's Dos Navigator with Dos Navigator Open Source Project

**Comment by determin1st** [comment]
less intuitive interface, doesnt track mouse move. first i went to panel settings to set my preferred view of a name+size but didnt find a way to span it to available size, sizes are fixed numbers
